### ISSUE TITLE
Save scaled/rounded/trimmed scores for training dataset predictions

### ISSUE THREAD

**Issue by bndgyawali** [EducationalTestingService/rsmtool#90]
Currently, `rsmtool` saves scaled/rounded/trimmed scores for test set predictions only. Also, save them for training set predictions.

**Comment by bndgyawali** [comment]
#91 resolved
